docs/dm/assets.md

```diff
@@ -87,62 +98,72 @@ This menu includes a plethora of options.
 
 ### Is a token
 
-Selecting this option classifies the asset as a token. 
-This will help you keep track of which assets are tokens and which are other assets.
+Selecting this option classifies the object as a token. 
+Tokens function fundamentally differently than other shapes.
+
+Tokens cannot pass through shapes with the 'blocks movement' option enabled. 
+Tokens are also effected by lighting.
+Check the [lighting and vision](/docs/dm/light-shadows/) document for more information on how lighting works in relation to tokens.
 
 ### Is invisible
 
-Selecting this option turns the asset invisible to your players.
-As the DM, you will still be able to see the asset.
+Selecting this option turns the shape invisible to your players.
+As the DM, you will still be able to see the shape.
 This can be useful for hiding future encounters or placing enemies in stealth. 
 
+Players with vision or edit access will still see the object with this enabled.
+There is currently no special indicator that shows it's invisible to other users. 
+
 ### Is locked
 
-Selecting this option locks the object in place.
+Selecting this option locks the shape in place.
 This can be useful for maps you don’t want to accidently move while playing.
+Press ```Cntrl + L``` to do this without opening the Edit asset menu.
 
 ### Show badge
```

Comment:
I'd like to see a bit more info on badges. As far as I've used them, they're incrementing numbers, but it would be helpful to know (and write down here) how exactly the numbers are counted. I.e.: one list, including every shape? one list for each location? one list for each type of asset? etc.

Honestly, I do not know the answer, but it should be easy to find/test out at least the basics (and then, of course, look into the code and/or ask its master).